build/kubernetes/coredns_clusterroles_patch.yaml

```diff
@@ -0,0 +1,9 @@
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - endpoints
+      - endpointslices
+      - services
+      - pods
+      - namespaces
```

Comment:
I think we might need to add in the `verbs` values as well...
```
rules:
- apiGroups:
  - ""
  resources:
  - endpoints
  - endpointslices
  - services
  - pods
  - namespaces
  verbs:
  - list
  - watch
```

Reply:
good catch. thanks